src/version.ts

```diff
@@ -1,2 +1,3 @@
+export const SDK_PACKAGE_NAME = 'npm: @sentry/capacitor';
```

Comment:
L: Other SDKs don't have the space.
```suggestion
export const SDK_PACKAGE_NAME = 'npm:@sentry/capacitor';

```